tests/resources/zfin/zfin_test_ids.yaml

```diff
@@ -0,0 +1,575 @@
+genotype: 
```

Comment:
as mentioned in comments last week;
This file is formatted differently than the existing test_id.yaml files.
for example please see:
https://github.com/monarch-initiative/dipper/blob/master/resources/test_ids.yaml

Reply:
Hmm - [that file](https://github.com/monarch-initiative/dipper/blob/master/resources/test_ids.yaml) doesn't seem to be valid YAML...

Reply:
there are lines longer than 80 cols. (see the Makefile for how we ignore them)

and there are two commas that need a space after them...

good catch